src_mod/core/filesystem.h

```diff
@@ -27,7 +27,7 @@
 
 #include "q_shared.h"
 #include "cvar.h"
-#include <zlib/unzip.h>
+#include "../zlib/unzip.h"
```

Comment:
Revert it back.
Compiler uses -I flag which sets "src_mod" dir as include directory.